.github/workflows/build.yml

```diff
@@ -91,8 +91,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Pin Terraform version
       uses: hashicorp/setup-terraform@v1
-      with:
-        terraform_version: 0.12.29
+      #with:
+      #  terraform_version: 0.12.29
```

Comment:
can be pinned to a version `0.13.3` or `0.13.5`?

Reply:
done...